README.md

```diff
@@ -105,9 +105,9 @@ export UPGRADE_STRATEGY=composer
 ```
 #### Composer update strategy
 Composer install|update uses two strategies.
-By default composer updates only lock file.
+By default composer updates lock file with packages.
 
-* To enable update composer with packages:
+* To enable update composer lock file only:
```

Comment:
If it's already `true` by default, why do have this line that says how to enable it?